app/views/layouts/head/_scripts.html.erb

```diff
@@ -1,12 +1,17 @@
-<!-- Scripts -->
-<%= javascript_include_tag 'application' %>
-<% if @ext_js.present?
-  Array.wrap(@ext_js).each do |js|
-    concat "  #{javascript_include_tag js}\n".html_safe
-  end
-end %>
-<% if @inline_js.present?
-  Array.wrap(@inline_js).each do |js|
-    concat "  <script>\n#{js}\n  </script>\n".html_safe
-  end
-end %>
+  <!-- Scripts -->
+  <%= javascript_include_tag 'application' %>
+  <% if @ext_js.present? || @inline_js.present? %>
+  <!-- Conditional -->
```

Comment:
You pointed this out to me yesterday but I thought it was a header for the @ext and @inline js objects.  What kind of things do you picture going inside this if statement in overrides?